src/state.ts

```diff
@@ -51,7 +51,7 @@ export class UiFeedbackState {
    * Get the current value for a given feedback ID
    * @param feedbackId Internal ID of the feedback
    */
-  get(feedbackId: string) {
+  get(feedbackId: string): void {
```

Comment:
Does this really need to be void?

Reply:
Ugh you're right, thanks for the review! Will change this tomorrow